frontend/src/layouts/__snapshots__/Header.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`Tenant Header Header CITZ background 1`] = `
 .c0 {
   background-color: #003366;
   border: none;
-  border-bottom: 2px solid $accent-color;
+  border-bottom: 2px solid;
```

Comment:
Hmm, are we sure the props.them.css.accentColor is working? Maybe it just isn't working in the snapshots? I would expect to see the color show up here.

Reply:
The snapshot would require mocking the global scss variables, which I don't believe I have